src/services/tree-sitter/index.ts

```diff
@@ -106,6 +137,7 @@ async function parseFile(
 	filePath: string,
 	languageParsers: LanguageParser,
 	rooIgnoreController?: RooIgnoreController,
+	addLineNumbers?: boolean,
```

Comment:
Is there a reason to not just always add them?

Reply:
> Is there a reason to not just always add them?

no, i just don't know how it effect another function

Reply:
let add start_line and end_line by default ?